### ISSUE TITLE
Support Angular RC6 / Breaking Change Type<T>

### ISSUE THREAD

**Issue by StephanHartmann** [DevExpress/devextreme-angular2#93]
Update to RC6 leads to following errors
````
node_modules\devextreme-angular2\node_modules\@angular\forms\src\directives.d.ts:44:38
Generic type 'Type<T>' requires 1 type argument(s).
node_modules\devextreme-angular2\node_modules\@angular\forms\src\directives.d.ts:48:47
Generic type 'Type<T>' requires 1 type argument(s).
node_modules\devextreme-angular2\node_modules\@angular\forms\src\form_providers.d.ts:13:37
Generic type 'Type<T>' requires 1 type argument(s).
````
see [changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)

**Comment by StephanHartmann** [comment]
ok, the errors are related to the referenced old angular forms module 0.3.0

Can you nevertheless update to RC6.

I think you are using some of  the now deprecated  functions.

**Comment by marcelgood** [comment]
Same issue here. This is currently preventing me from going to RC6. Please release and update for RC6.

**Comment by Aden-git** [comment]
Hello,

Currently we are working on update. We recommend you use angular RC5 as a workaround until we do not update our project.

**Comment by StephanHartmann** [comment]
Updating to RC6 also means you are building ngModules, right?

Do you have a plan when RC6 version will be released?
It's not about to nail you on a special date ,
I know that this not really possible.

But a estimated timeframe like "next week, when no new blocking bugs appear" would be great

**Comment by Aden-git** [comment]
Hello,

You are right about ngModules. We support it in the context of an update to RC6.

As for the date of the release with RC6 support, we are currently working on it and will do our best to finish it as soon as possible.

**Comment by isaacplmann** [comment]
PR #102 should take care of the Type<T> errors.  It doesn't port everything to NgModules, but at least it compiles in RC6.

**Comment by Aden-git** [comment]
We have released version 16.1.7-alpha with support for Angular 2.0.
